cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
+  "snippet_name": "MySnippet",
+  "project_name": "NetSquid-{{ cookiecutter.snippet_name }}",
+  "project_slug": "{{ cookiecutter.project_name|lower|replace(' ', '-') }}",
+  "project_folder": "{{ cookiecutter.project_slug|replace('-', '_') }}",
+  "description": "Write some description of the package",
+  "author_name": "Name LastName",
+  "author_email": "name@email.com",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "url": "https://github.com",
```

Comment:
would make it more clear this should ideally point to their repository e.g. "https://github.com/username/mysnippet"

Reply:
Done